packages/kbn-shared-ux-components/src/page_template/solution_nav_avatar/solution_nav_avatar.scss

```diff
@@ -0,0 +1,14 @@
+.kbnPageTemplateSolutionNavAvatar {
+  @include euiBottomShadowSmall;
+
+  &--xxl {
+    @include euiBottomShadowMedium;
+    @include size(100px);
```

Comment:
what is this mixin? I haven't managed to find its definition in `eui` codebase. is it needed?